composites/Plugin/ContentAnalysis/actions/contentAnalysis.js

```diff
@@ -119,3 +122,32 @@ export function updateReadabilityResult( result ) {
 		result: result,
 	};
 }
+
+/**
+ * An action creator for setting the overall score for a readability result.
+ *
+ * @param {Object} results The overall score.
+ *
+ * @returns {Object} A set overall score action.
+ */
+export function setOverallScoreReadability( overallScore ) {
```

Comment:
setOverallReadabilityScore is easier to read (also for the type below and the SEO ones)